doc/api/c++/common/libs.md

```diff
@@ -4,39 +4,9 @@ Libs -- Wrappers for C libraries functions.
 # LIBRARY
 MeasurementKit (libmeasurement-kit, -lmeasurement-kit).
 
-# SYNOPSIS
-```C++
-#include <measurement_kit/common.hpp>
-
-using namespace measurement_kit::common;
-
-// Suppose you want to check whether the Poller constructor deals
-// well with `event_base_new()` returning `nullptr`, then:
-
-Libs libs;
-libs.event_base_new = []() { return nullptr; };
-bool exc = false;
-try {
-    Poller poller(&libs);
-} catch (std::bad_alloc &) {
-    exc = true;
-}
-if (!exc) { /* test failed, do something */ }
-```
-
```

Comment:
Here I would keep a small synopsis example showing with a fake class how to access the global pointer to the `Libs` object. You can see, for example, how this functionality is used by the `Poller`.